app/src/main/java/com/theopensourcefamily/chessclock/BoardState.kt

```diff
@@ -0,0 +1,6 @@
+package com.theopensourcefamily.chessclock
+
+sealed class BoardState
+object Stopped : BoardState()
+object WhiteRunning : BoardState()
```

Comment:
I'll answer your PR question regarding the Enum vs Sealed here if you don't mind 😸 
The idea behind this class is to be the only source of truth about what's actually shown in the screen. That means we won't be able to render everything with a state without params, here's why:
Every second (for example while whites are running) a new state is supposed to arrive to be rendered, and that state needs to have the clocks info: whitesCount, blacksCount, ammount of turns, etc... and that parameters need to be part of the data class extending the sealed class.
This atomic states make the view more robust, as you cannot fall in inconsistent states, like running + showing the play button, since it will be only displayed in the pause and stopped states.
So to sum up: everytime the screen needs to change, a new state has to be rendered, and that state has to fyully contain all the necessary info (like a viewmodel). There are some exceptions that are not states (like snackbar, or toasts) that need to be addressed in other ways, but we'll discuss it in another PR! :D)

Reply:
Thank you very much for this detailed explanation! You convinced me since second 1 :)